### ISSUE TITLE
Better UI for landing page

### ISSUE THREAD

**Issue by ADARSHsri2004** [anjalit03/Dazzle-and-Delight#26]
The navbar can be better, footer section is not aligned and UI can be better. I can help in solving this.

**Comment by ADARSHsri2004** [comment]
Please assign to fix this . @anjalit03

**Comment by anjalit03** [comment]
yes I'm assigning this to you! someone else is already working on navbar so you can work on footer 

![Screenshot 2024-10-28 205140](https://github.com/user-attachments/assets/06094e6f-119d-43e3-baa7-acbb9aead7cb)

this is the footer it should be consistent on all pages. thankyou !

**Comment by rishishanbhag** [comment]
please assign this to me too under hacktoberfest . The footer still seems to not be in shape.
